README.md

```diff
@@ -25,76 +25,40 @@ return [
 ```
 
-Now, it is time to choose you the events you are interested in. 
+SQS, SNS and EventBridge can now be used with Symfony Messenger.
 
-## Configuration
+## Usage
+
+Symfony Messenger dispatches messages. To create a message, follow the [Symfony Messenger documentation](https://symfony.com/doc/current/messenger.html#creating-a-message-handler).
 
-This bundle has Symfony Messenger Transports to publish messages and Consumers
-to receive Lambda events from AWS. All Transports are configurable with a DSN and 
-the sections below will show you some examples. They will all follow the normal 
-Symfony pattern: 
+To configure **where** messages are dispatched, all the examples in this documentation are based on [the example from the Symfony documentation](https://symfony.com/doc/current/messenger.html#transports-async-queued-messages): 
 
 ```yaml
 # config/packages/messenger.yaml
 
 framework:
     messenger:
         transports:
-            async: 
-                dsn: '%env(MESSENGER_TRANSPORT_DSN)%'
+            async: '%env(MESSENGER_TRANSPORT_DSN)%'
         routing:
              'App\Message\MyMessage': async
 ```
 
-To consume messages that has been on the queue, you need to use a *consumer* service.
-
 ### SQS
 
-The [SQS](https://aws.amazon.com/sqs/) service is a queue that works similar to
-RabbitMQ. The AWS console lets you create a SQS queue as a "normal queue" or a
-"FIFO queue". 
-
-> Note that environment variables `AWS_REGION`, `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`
-do always exist on Lambda. The AWS client will read `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`
-automatically. 
-
-#### Normal queue
-
-```yaml
-# config/packages/messenger.yaml
-
-framework:
-    messenger:
-        transports:
-            my_sqs: 
-                dsn: 'https://sqs.us-east-1.amazonaws.com/123456789/my-queue'
+The [SQS](https://aws.amazon.com/sqs/) service is a queue that works similar to RabbitMQ. To use it, create a SQS queue and set its URL as the DSN:
 
-services:
-    my_sqs_consumer:
-        class: Bref\Symfony\Messenger\Service\Sqs\SqsConsumer
-        public: true
-        arguments:
-            - '@Bref\Symfony\Messenger\Service\BusDriver'
-            - '@messenger.routable_message_bus'
-            - '@Symfony\Component\Messenger\Transport\Serialization\SerializerInterface'
-            - 'my_sqs' # Same as transport name
+```dotenv
+MESSENGER_TRANSPORT_DSN=https://sqs.us-east-1.amazonaws.com/123456789/my-queue
 ```
 
-Now, let's create our Lambda handler (for example `bin/consumer.php`):
-
-```php
-<?php declare(strict_types=1);
+That's it, messages will be dispatched to that queue.
 
-require dirname(__DIR__) . '/config/bootstrap.php';
+> Note: when running Symfony on AWS Lambda, it is not necessary to configure credentials. The AWS client will read [`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html#configuration-envvars-runtime) automatically.
```

Comment:
I am not sure this should be changed or not. However, this is not correct. 

I am pretty sure `AWS_SECRET_ACCESS_KEY` is not available (Or i missed it when I looked).  However `AWS_SESSION_TOKEN` is available and will be picked up automatically.

Reply:
It actually reads the 3 variables, but that's a detail. I've reworded the sentence to simplify it and avoid the details.